packages/js/src/initializers/search-appearance.js

```diff
@@ -75,7 +75,7 @@ export default function initSearchAppearance() {
 				{ imagePortals.map( ( portal ) => {
 					return ( <ImageSelectPortal
 						key={ portal.id }
-						label={ portal.dataset.reactImagePortalLabel || __( "Social default image", "wordpress-seo" ) }
+						label={ portal.dataset.reactImagePortalLabel || __( "Social image", "wordpress-seo" ) }
```

Comment:
If `Social image` is now the default, we don't need to specify that exact value anymore in `admin/views/tabs/metas/paper-content/front-page-content.php:39`